### PR TITLE
Page `ViewTopicDelegations`

### PR DESCRIPTION
**Pull request by fisx**
(not really showing anything, but the mechanics have made some progress.)

### CODE REVIEW

File: exec/RenderHtml.hs [hunk 1]
```diff
@@ -128,8 +131,34 @@ instance Arbitrary (ToHtmlSpecial ViewIdea) where
         p = pure Nothing
 
 
--- newtypes for one 'ViewIdea' sample per phase.  the '_' in the mae are allowed here because they
--- make it clear where teh page type ends and other information begins.
+-- * pages with many modes
+
+-- newtypes for pages that have different views.  for instance, 'ViewIdea' has one sample per phase.
+-- the '_' in the mae are allowed here because they make it clear where teh page type ends and other
```
Comment (Mikolaj) [inline exec/RenderHtml.hs]: typos: mae, teh

**Comment by Mikolaj** [comment]
Very shallow LGTM.

**Comment by fisx** [comment]
@andorp could you take a look and assign back to me or merge?  thanks!

File: src/Frontend/Page/Topic.hs [hunk 2]
```diff
@@ -97,43 +98,53 @@ instance ToHtml EditTopic where
...
-            when (phase < PhaseResult) .
-                a_  [class_ "btn-cta heroic-cta", href_ . U.Space space $ U.CreateTopicDelegation topicId] $ do
-                    i_ [class_ "icon-bullhorn"] nil
-                    "Stimme Beauftragen"
-            div_ [class_ "heroic-tabs"] $ do
-                tabLink topic tab TabAllIdeas
-                when (phase >= PhaseVoting) $ tabLink topic tab TabVotingIdeas
-                when (phase >= PhaseResult) $ tabLink topic tab TabWinningIdeas
-                tabLink topic tab TabDelegation
+        assert (tab /= TabDelegation) $ viewTopicHeaderDiv topic tab
```
Comment (andorp) [inline src/Frontend/Page/Topic.hs]: Generally I don't like asserts in code, but for the time being I can live with it.

**Comment by andorp** [comment]
Issue created: #254

**Comment by andorp** [comment]
Otherwise LGTM

**Pull request merged**